hrm-service/src/csam-report/csam-report-data-access.ts

```diff
@@ -1,54 +1,75 @@
 import { db } from '../connection-pool';
-import { insertCSAMReportSql, NewCSAMReportRecord } from './sql/csam-report-insert-sql';
+import { insertCSAMReportSql } from './sql/csam-report-insert-sql';
 import {
   selectSingleCsamReportByIdSql,
   selectCsamReportsByContactIdSql,
 } from './sql/csam-report-get-sql';
-import { updateContactIdByCsamReportIdsSql } from './sql/csam-report-update-sql';
+import { updateContactIdByCsamReportIdsSql, updateAcknowledgedByCsamReportIdSql } from './sql/csam-report-update-sql';
 // eslint-disable-next-line prettier/prettier
 import type { ITask } from 'pg-promise';
 
-export type CSAMReportRecord = NewCSAMReportRecord & {
-  id: number;
+export type NewCSAMReport = {
+  reportType: 'counsellor-generated' | 'self-generated';
+  acknowledged: boolean;
+  twilioWorkerId?: string;
+  csamReportId?: string;
+  contactId?: number;
 };
 
-export type CreateCSAMReport = Omit<NewCSAMReportRecord, 'accountSid'>;
+export type CSAMReport = NewCSAMReport & {
+  id: number;
+  accountSid: string;
```

Comment:
Do we need this as part of the type? We should only ever be dealing with reports from one account in HRM anywat